apps/combiner/src/server.ts

```diff
@@ -29,17 +29,17 @@ import { pnpQuota } from './pnp/endpoints/quota/action'
 import { pnpSign } from './pnp/endpoints/sign/action'
 import {
   CachingAccountService,
-  ContractKitAccountService,
   MockAccountService,
+  ViemAccountService,
 } from './pnp/services/account-services'
 import { NoQuotaCache } from './utils/no-quota-cache'
 
 require('events').EventEmitter.defaultMaxListeners = 15
 
-export function startCombiner(config: CombinerConfig, kit?: ContractKit) {
+export function startCombiner(config: CombinerConfig, viemClient?: WalletClient) {
   const logger = rootLogger(config.serviceName)
 
-  kit = kit ?? getContractKitWithAgent(config.blockchain)
+  viemClient = viemClient ?? getWalletClientWithAgent(config.blockchain)
```

Comment:
Nit: consider naming all these "ClientAccountService" etc. to be consistent with the Signer server.ts file